README.md

```diff
@@ -1,10 +1,55 @@
-﻿# Bot
+# Clockify NLP Bot
 
-Bot Framework v4 empty bot sample.
+**Clockify NLP Bot** is a [Bot Framework](https://dev.botframework.com) bot developed to act as a timesheet assistant for the busy and the lazy among us.
+It reminds proactively people to fill in their time when EOD approaches, and it accepts natural language for creating time entries directly from chat.
 
-This bot has been created using [Bot Framework](https://dev.botframework.com), it shows the minimum code required to build a bot.
+<img src="https://user-images.githubusercontent.com/11543564/133216495-aa6feb9d-c30e-45f4-aeed-0329e0f75585.png" width="700">
 
-## Prerequisites
+## How to use it
+It's simple, just open a chat with the bot either on Teams or Telegram.
+
+👉 [Chat on Telegram](http://t.me/clockify_telegram_bot)  
+👉 [Chat on Teams](https://teams.microsoft.com/l/chat/0/0?users=28:d7dfef09-5ad7-4e1c-ac47-a3899bf1964c)
+
+
+## Features:
+
+**Proactive reminders**
+It will assist you to complete your due diligence. If you have less than 6 hours 
+reported on your Clocki (no matter how you put them in) it sends you a reminder.
```

Comment:
typo: Clockify

Reply:
not a typo :)

Reply:
but if you think it is I should probably change it anyway